frontend/src/RadioInput.js

```diff
@@ -0,0 +1,23 @@
+import classNames from 'classnames'
+
+function RadioInput(props) {
+  const { field, id, label } = props
```

Comment:
Ez lehetne useField-es, mint ahogy a többi Formikos komponensnél is csináltátok.